entries/git-workflow-guidelines/index.md

```diff
@@ -0,0 +1,79 @@
+
+--- issue templates, in another post? "Furthermore, we have implemented a couple of issue templates to follow."
+
+### What is Git?
+
+https://github.com/git-guides
+
+This could also be the place for explaining how Git tracks files and local vs. remote repositories?
+
+### Branching Strategy: GitHub Flow
+
+Different branching strategies can be applied, depending on the project, the team, and the organisation (as well as preferences). We have chosen to follow a GitHub Flow approach. To read about the drivers for this decision, see the [GitHub flow decision post](../why-github-flow/index.md).
+
+This approach has, generally speaking, the following steps:
+
+1. Create a branch from main (e.g., a branch introducing a new feature or fixing a bug)
+2. Change or add files to implement the feature
+3. Create a pull request and request reviews from your collaborator (and discuss the changes in the PR)
+4. Collaborators review the changes and might suggest changes
+5. Implement suggested changes (if any)
+6. When the changes has been approved, merge the feature branch into the main branch
+7. Delete the feature branch
+
+![Example of the GitHub flow showing the process of creating and merging a feature branch](/entries/images/github-flow.png)
+
+There are a few points about this kind of workflow, we would like to emphasise:
+
+- **The main branch contains production-ready code**: In this workflow, branches are created to work on new features, bug fixes, or the like and merged into the main branch as soon as the work is completed and has been reviewed. After the merge with main, the branch is deleted.
+- **One branch for one purpose**: Create a branch for one specific purpose. This allows your collaborators to easily get an overview of the ongoing work. Furthermore, the shorter the changes, the faster it will be reviewed and merged to main.
+- **Collaboration happens in the PRs**: Better work is created when you are not working in a silo. During PRs, you'll have a conversations about the proposed changes and your collaborators might have some ideas for improvement. PRs is a chance to utilise each other's experiences and expertise
+- **The change does not have to be fully implemented before creating a PR**: When you create a PR, you communicate to your collaborators what you are working on. In this [talk](https://www.youtube.com/watch?v=vCwuZfK0VG4), it is recommended to create your PRs early and not wait until your work is complete, for two reasons: 1) it helps prevent that someone else does not start doing the same work as you are currently working on, and 2) you can start the discussion of the changes you are currently implementing. Remember to mark the PR as a [draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request#converting-a-pull-request-to-a-draft), if it's still a work in progress.
```

Comment:
you have a double-negative in point 1.  prevent that they don't start

Reply:
Nice catch. Thanks!